.gitignore

```diff
@@ -32,3 +32,4 @@ yarn-error.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.vs/slnx.sqlite
```

Comment:
Since VS is not recommended editor for this project, these .vs config files should not be included at all. I recommend removing this line and removing the .vs/ folder from the commit entirely
```suggestion
```